core/modulesource_test.go

```diff
@@ -47,3 +51,390 @@ func TestGitModuleSourceSymbolic(t *testing.T) {
 		})
 	}
 }
+
+func TestMatchVersion(t *testing.T) {
```

Comment:
Just wondering if this test shouldn't be in a `core/modulerefs_test.go`?
This is mostly about conventions and keeping the test as close as possible to the code itself, but also helps IDEs like Goland that groups test files with their source code
<img width="205" alt="image" src="https://github.com/user-attachments/assets/f7f165ab-3c36-43c1-bc3e-4575cca4b897" />

Reply:
Done in [b06969c](https://github.com/dagger/dagger/pull/10309/commits/b06969c102812b7feede5d7a4850acc20e368101)